ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -196,6 +196,48 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-single-node
+- as: e2e-network-mtu-migration-ovn-ipv4
```

Comment:
don't you need the DEV_SCRIPTS config here to use NETWORK_TYPE=OVNKubnernetes ?

Reply:
No, in this case OVN config is handled through task `ovn-conf` within the `openshift-e2e-gcp-ovn` workflow. 

I will set them all as optional.

Reply:
gotcha. I saw it below in the e2e-network-mtu-migration-ovn-ipv6 job which made me think
it was missing here. But, now I realize you could also do this same thing below I think and not
have to include DEVSCRIPTS_CONFIG if you use the baremetalds-e2e-ovn-ipv6 workflow.

NBD though. either works fine.

Reply:
Ok, I missed that one. I will try it out.

Reply:
👀

Reply:
what examples do you have? looking at this recent [one](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24734/rehearse-24734-pull-ci-openshift-cluster-network-operator-master-e2e-network-mtu-migration-ovn-ipv4/1509163839768236032), it looks like the must-gather step (after the tests)
failed because of some imagePull issue.